Question14.java

```diff
@@ -0,0 +1,26 @@
+class Solution {
+    public String longestCommonPrefix(String[] strs) {
+        int m = 0;
```

Comment:
It is a good idea to add simple comments such as :
`// maximum common prefix so far`

When problem gets hard, you'll spend most of your time reading your own code (again and again), so any visual aid you leave for yourself will save you time and might as well be critical.